FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,21 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		self.client.get(from: self.url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case .failure(_):
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
The `Error` is declared under`RemoteFeedLoader`, so `RemoteFeedLoader` can be omitted in this expression.

```suggestion
				completion(.failure(Error.connectivity))
```